library/src/main/java/com/flaviofaria/kenburnsview/KenBurnsView.java

```diff
@@ -25,14 +25,17 @@
 import android.util.AttributeSet;
 import android.widget.ImageView;
 
+import com.burningthumb.premiervideokiosk.DrawOnImageView;
+
 /**
  * {@link ImageView} extension that animates its image with the
  * <a href="http://en.wikipedia.org/wiki/Ken_Burns_effect">Ken Burns Effect</a>.
  * @author Flavio Faria
  * @see Transition
  * @see TransitionGenerator
  */
-public class KenBurnsView extends ImageView {
```

Comment:
You are using a bad indentation, it seems like C# is not the standard used in Kotlin!